tests/map/options.test.ts

```diff
@@ -0,0 +1,41 @@
+import { MapOptions } from '../../src/map/options';
+
+import { default as setupJSDOM } from '../jsdom';
+import { assert } from 'chai';
+
+const DUMMY_PROPERTIES = {
+    first: {
+        values: [],
+    },
+    second: {
+        values: [],
+    },
+};
+const DUMMY_CALLBACK = () => {
+    return { top: 0, left: 0 };
+};
+
+describe('MapOptions', () => {
+    before(() => {
+        setupJSDOM();
+    });
+
+    it('validate new values for properties');
+
+    it('creates HTML elements with unique id');
+
+    it('can save settings');
+
+    it('can apply saved settings');
```

Comment:
I'm not so familiar with this status -- can you explain what the successive `it(...)` statements do?

Reply:
So the way mocha works is that test cases are introduced as calls to the `it` function, with a first argument describing the test case, and the second argument executing the test itself. If there is no second argument, as here, the test is skipped and considered "to be written later". 

I added these mostly as a TODO list for myself, but I can also keep them out of the code for now.

Reply:
It might be best to either omit in this PR or write the coinciding tests

Reply:
Removed, I'll send another PR implementing these tests